CHANGELOG.md

```diff
@@ -5,11 +5,16 @@ It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
 ## Kiwix-JS v2.8.0
+
 Released on *TODO*
 
-Detailed changelog : TODO
+* NEW: Intelligently select the best Storage API for storing settings between sessions
+* UPDATE: Better handling of case in title search, returning more accurate search results
```

Comment:
True, but your PR deserves another line like : 
`NEW: display search results as they are discovered instead of waiting to have all of them before displaying them`
(you might improve the wording)